testsuite/integration-arquillian/tests/base/src/test/resources/testrealm.json

```diff
@@ -501,6 +500,18 @@
         "profile",
         "email"
       ]
+    },
+    {
+      "clientId": "direct-grant-1",
```

Comment:
Is it possible to avoid changes in this file `testrealm.json` and instead create clients programatically in your test? The `testrealm.json` is used by many other tests and it doesn't need the overhead of 2 additional clients created just for the purpose of single test.

Clients could be created for example in the method `configureTestRealm` , which is already overriden for the `UserSessionLimitsTest`